go.mod

```diff
@@ -1,10 +1,13 @@
 module github.com/aws-controllers-k8s/prometheusservice-controller
 
 go 1.17
+
 require (
 	github.com/aws-controllers-k8s/runtime v0.18.4
```

Comment:
We just released v0.19.3. Can you please send a PR to the amp controller repository making just a runtime update?